lib/class-wp-rest-menu-items-controller.php

```diff
@@ -479,21 +479,23 @@ protected function prepare_item_for_database( $request ) {
 			}
 		}
 
+		$error = new WP_Error();
```

Comment:
This is not needed, but makes debugging easier, as you can see multiple errors in REST API response.